Add password successful feedback

antonlindstrom opened the pull request:
This adds some feedback when entering the password on the command line.
When the password is entered and supplied through stdin (and stdout is a
terminal) then the a message saying `password is correct` if correct is
printed.

The pull request aims to fix #438 

I'm not sure if it's the correct place to output in the `OpenRepository` function
or that my assumption is correct in when it should be outputted.

Thanks!

antonlindstrom committed:
Add password successful feedback

This adds some feedback when entering the password on the command line.
When the password is entered and supplied through stdin (and stdout is a
terminal) then the a message saying `password is correct` if correct is
printed.

rawtaz commented:
I have a thought on this: Why would you not output the text just because the password happens to come from an environment variable or password file, rather than the terminal? I mean, it's still a password, and whether it's correct or not is still relevant. In short, I think you should output the text regardless of the source of the password.

antonlindstrom committed:
Output password successful on terminal stdout

This removes the conditions that checks if the password is supplied
through environment variable or file and outputs password is successful
on terminal and when --quiet is not supplied.

fd0 reviewed (approved):
Nice, that was easy.
